setup.sh

```diff
@@ -2,6 +2,13 @@
 # Portable way to get current directory
 SPACK_STACK_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+hostname=$(hostname)
+case $hostname in
+  alogin*)
```

Comment:
That hostname is very unfortunate. I can totally see other platforms using something like this, too ... but for now we don't have any other one.

Reply:
We can use `hostname --fqdn`. I'll be out the next couple days but I'll update this when I get the chance. With that it can be alogin*ncep.noaa.gov or something like that.

Reply:
We can add that when we need it - for now we're ok.